testng.xml

```diff
@@ -1,10 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
 
-<suite name="Yandex Test">
-    <test name="Login Form Tests">
+<suite name="Regression Tests"
+       thread-count="3" parallel="tests" verbose="3">
+
+    <test name="Magento ChromeBrowser Test">
+        <parameter name="browser" value="chrome"/>
+        <classes>
+            <class name="tests.LogInTest"/>
+            <class name="tests.AccountCreationTest"/>
+            <class name="tests.AddressBookTest"/>
+            <class name="tests.StoreTests"/>
+        </classes>
+    </test>
+
+    <test name="Magento FirefoxBrowser Test">
```

Comment:
remove redundant and try this:

```
<suite name="Regression Tests"
       thread-count="4" parallel="classes">

    <test name="Magento Test">
        <classes>
            <class name="tests.LogInTest"/>
            <class name="tests.AccountCreationTest"/>
            <class name="tests.AddressBookTest"/>
            <class name="tests.StoreTests"/>
        </classes>
    </test>
</suite>
```

then right click on testng.xml and Run

![image](https://user-images.githubusercontent.com/62067083/211781338-86d84b23-00b1-4ccf-b8c8-818f6e96967a.png)